### DISCUSSION TITLE
Implement missing properties and methods for the drawing context

### DISCUSSION THREAD

**Discussion by dmitry-medvedev** [alternetsoft/AlternetUI#121] [Ideas]

**Comment by dmitry-medvedev** [comment]
[methDW.csv](https://github.com/alternetsoft/AlternetUI/files/10830146/methDW.csv)



[propDW.csv](https://github.com/alternetsoft/AlternetUI/files/10830147/propDW.csv)

**Comment by generalloki** [comment]
Also see https://github.com/alternetsoft/AlternetUI/issues/7
for updated methDW and popDW files

**Comment by generalloki** [comment]
We started a process of making UI more compatible with WinForms.
New methods which are present in WinForms graphics are being added to the UI.
Also some enums will be renamed to be more compatible with WinForms.
I will post an updates here.

**Comment by neoxeo** [comment]
@generalloki

Update files to compare same/different methods/properties between AlternetUI Drawing version 0.9.410.0 and .Net8.0 Drawing : 

[Properties_Drawing_Net8.0_2023.12.02_070609.xlsx](https://github.com/alternetsoft/AlternetUI/files/13533561/Properties_Drawing_Net8.0_2023.12.02_070609.xlsx)
[Methods_Drawing_Net8.0_2023.12.02_070609.xlsx](https://github.com/alternetsoft/AlternetUI/files/13533562/Methods_Drawing_Net8.0_2023.12.02_070609.xlsx)

**Comment by generalloki** [comment]
@neoxeo , Thanks! 

I beleive we will support as much WiinForms Graphics methods as we can. I will post here our progress.

**Comment by neoxeo** [comment]
@generalloki 

Here is an update with Properties, Methods and Events for commit version https://github.com/alternetsoft/AlternetUI/commit/144b44c1d94a985e93a5b3510fdb322202648f3c :
[Events_Drawing_Net8.0_2023.12.10_130058.xlsx](https://github.com/alternetsoft/AlternetUI/files/13627640/Events_Drawing_Net8.0_2023.12.10_130058.xlsx)
[Properties_Drawing_Net8.0_2023.12.10_130058.xlsx](https://github.com/alternetsoft/AlternetUI/files/13627641/Properties_Drawing_Net8.0_2023.12.10_130058.xlsx)
[Methods_Drawing_Net8.0_2023.12.10_130058.xlsx](https://github.com/alternetsoft/AlternetUI/files/13627642/Methods_Drawing_Net8.0_2023.12.10_130058.xlsx)

**Comment by generalloki** [comment]
Good Job! Thanks!

**Comment by generalloki** [comment]
I converted this to discussion. Individual properties and methods in graphics context that we need to implement will be issues. And here we can speak about the topic in general.
